tests.html

```diff
@@ -42,5 +42,6 @@
         These unit tests should be run with Firefox.<br/>
         TODO : find out why they do not all run on Chromium
```

Comment:
I think you can remove the TODO, as this PR fixes it :-)
It also might be the right place to explain how to run the tests on Chromium/Chrome : either with the command-line argument, or through a local webserver